### PR TITLE
Always use native pgrep on darwin

### PR DESCRIPTION
**Pull request by inversion**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #3262 
| License       | MIT

### CODE REVIEW

**Comment by Unitech** [comment]
how did you ended up having another pgrep on your system?

**Comment by inversion** [comment]
`brew install proctools` installs another pgrep in `/usr/local/bin` - please see #3262

**Comment by chris-allnutt** [comment]
Experiencing this as well.  Took me forever to figure out why it was breaking.  I hope this can be merged in at some point.

**Comment by sonnenhaft** [comment]
pls merge

File: lib/Watcher.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ var chokidar = require('chokidar');
 var p        = require('path');
 var util     = require('util');
 var log      = require('debug')('pm2:watch');
+var crypto   = require('crypto');
```
Comment (soyuka) [inline lib/Watcher.js]: not sure that this code should be here, could you check?

**Pull request closed without merging**